client/scripts/controllers/server/comments.ts

```diff
@@ -28,6 +26,17 @@ export const modelFromServer = (comment) => {
     ? comment.OffchainAttachments.map((a) => new OffchainAttachment(a.url, a.description))
     : [];
 
+  const versionHistory = comment.version_history.map((v) => {
+    let history;
+    try {
+      history = JSON.parse(v || '{}');
+      history.timestamp = moment(history.timestamp);
+    } catch (e) {
+      console.log(e);
+    }
+    return history;
+  });
+
```

Comment:
Trying to understand what is happening here

Why is every entry of version_history being overwritten with a new `timestamp`?

Reply:
version_history comes off the server as a stringified object. This parses the object and converts the timestamp to a moment.

Previously, these steps had to be done each time you wanted to interact with version_history in a given component or page. Now they're done in a centralized manner, up front.